lib/ar_helper.rb

```diff
@@ -7,7 +7,7 @@
 require "application_record"
 Dir[File.expand_path("app/models/**/*.rb", app_root)].each { |f| require f }
 
-app_env = ENV["TOPOLOGICAL_INVENTORY_ENV"] || "development"
+app_env = ENV["ENV"] || "development"
```

Comment:
RAILS_ENV maybe? ... this particular env var now feels too general.

Reply:
:+1: done